packages/wdio-runner/src/index.ts

```diff
@@ -102,6 +102,9 @@ export default class Runner extends EventEmitter {
          * autocompile after parsing configs so we support ES6 features in tests with config driven by users
          */
         if (args.autoCompileOpts?.autoCompile) {
+            if (!fs.existsSync(args.autoCompileOpts.tsNodeOpts.project))
+                log.error('Path to "tsconfig.json" is incorrect. Is it correclty set in config -> tsNodeOpts ? ')
```

Comment:
Let's throw an error here because if someone really defines this property it should point to a correct file